packages/swayze/lib/src/widgets/table_body/gestures/table_body_gesture_detector.dart

```diff
@@ -314,6 +314,15 @@ class _TableBodyGestureDetectorState extends State<TableBodyGestureDetector> {
         return Listener(
           behavior: HitTestBehavior.translucent,
           onPointerDown: (details) {
+            final isMouse = details.kind == PointerDeviceKind.mouse;
+            final isPrimary = details.buttons == kPrimaryButton;
```

Comment:
We should probably use `kPrimaryMouseButton` here since we're referring to a mouse, right?

```suggestion
            final isPrimary = details.buttons == kPrimaryMouseButton;
```

Reply:
Good question. `kPrimaryButton` includes `https://api.flutter.dev/flutter/gestures/kPrimaryButton-constant.html` according to the docs. I think it's fine right? Tapping the primary button on a touch device, stylus or mouse should start a new selection?

Reply:
Yes, but just below we're checking `!isMouse || isPrimary`, so I think it'd be clearer if it were `!isMouse || isPrimaryMouseButton`. This way, I think it's more explicit that we either accepts a pointer down from anything other than a mouse, or a primary mouse click